source/js/components/share-button-group/share-button-group.jsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import classNames from "classnames";
+import copyToClipboard from "../../copy-to-clipboard";
+
+export default class ShareButtonGroup extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      linkCopied: false
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.whenLoaded) {
+      this.props.whenLoaded();
+    }
+  }
+
+  renderFacebookButton() {
+    let label =
+      this.props.version === `mini` ? (
+        <span class="sr-only">Share by Facebook</span>
+      ) : (
+        `Faceook`
+      );
+
+    let link = this.props.link ? this.props.link : ``;
+
+    return (
+      <a
+        class="btn btn-secondary btn-share facebook-share"
+        href={`https://www.facebook.com/sharer/sharer.php?u=https://${link}`}
+      >
+        {label}
+      </a>
+    );
+  }
+
+  renderTwitterButton() {
+    let shareText = this.props.shareText ? this.props.shareText : ``;
+    let label =
+      this.props.version === `mini` ? (
+        <span class="sr-only">Share by Twitter</span>
+      ) : (
+        `Twitter`
+      );
+
+    return (
+      <a
+        class="btn btn-secondary btn-share twitter-share"
+        href={`https://twitter.com/intent/tweet?text=${encodeURIComponent(
+          shareText
+        )}`}
+      >
+        {label}
+      </a>
+    );
+  }
+
+  renderEmailButton() {
+    let shareText = this.props.shareText ? this.props.shareText : ``;
+    let label =
+      this.props.version === `mini` ? (
+        <span class="sr-only">Share by email</span>
+      ) : (
+        `Email`
+      );
+
+    return (
+      <a
+        class="btn btn-secondary btn-share email-share"
+        href={`mailto:?&body=${encodeURIComponent(shareText)}`}
+      >
+        {label}
+      </a>
+    );
+  }
+
+  renderLinkButton() {
+    let label = this.state.linkCopied ? `Copied` : `Copy`;
+    label =
+      this.props.version === `mini` ? (
+        <span class="sr-only">{label} page link</span>
+      ) : (
+        label
+      );
+
+    let classes = classNames(`btn btn-secondary btn-share link-share`, {
+      copied: this.state.linkCopied
+    });
+
+    let onClickHandler = event => {
```

Comment:
Does this need to be inlined, or will this work as a class level function, so we can use `onClick={ () => this.handleLinkClicked() }`?